### ISSUE TITLE
Unable to start on Ubuntu 22.04 b/c of libssl1 dependency

### ISSUE THREAD

**Issue by giantcow** [fosslife/devtools-x#29]
Unable to start dev-tools because of an libssl1 dependency which is introduced because of the `allowList`: `all` config here: https://github.com/fosslife/devtools-x/blob/master/src-tauri/tauri.conf.json#L50

```
# kayo @ Elpis in ~ [22:42:52] C:127
$ sudo dpkg -i ~/Downloads/dev-tools_1.7.1_amd64.deb
(Reading database ... 242407 files and directories currently installed.)
Preparing to unpack .../dev-tools_1.7.1_amd64.deb ...
Unpacking dev-tools (1.7.1) over (1.7.1) ...
Setting up dev-tools (1.7.1) ...
Processing triggers for mailcap (3.70+nmu1ubuntu1) ...
Processing triggers for gnome-menus (3.36.0-1ubuntu3) ...
Processing triggers for desktop-file-utils (0.26-1ubuntu3) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...

# kayo @ Elpis in ~ [22:43:07] 
$ source ~/.zshrc

# kayo @ Elpis in ~ [22:43:13] 
$ dev-tools
dev-tools: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory

# kayo @ Elpis in ~ [22:48:05] 
$ sudo dpkg -l | grep 'libssl\|openssl'
ii  libgnutls-openssl27:amd64                      3.7.3-4ubuntu1.1                        amd64        GNU TLS library - OpenSSL wrapper
ii  libssl-dev:amd64                               3.0.2-0ubuntu1.6                        amd64        Secure Sockets Layer toolkit - development files
ii  libssl3:amd64                                  3.0.2-0ubuntu1.6                        amd64        Secure Sockets Layer toolkit - shared libraries
ii  libssl3:i386                                   3.0.2-0ubuntu1.6                        i386         Secure Sockets Layer toolkit - shared libraries
ii  openssl                                        3.0.2-0ubuntu1.6                        amd64        Secure Sockets Layer toolkit - cryptographic utility
ii  perl-openssl-defaults:amd64                    5build2                                 amd64        version compatibility baseline for Perl OpenSSL packages

# kayo @ Elpis in ~ [22:49:36] C:130
$ uname -a
Linux Elpis 5.15.0-48-generic #54-Ubuntu SMP Fri Aug 26 13:26:29 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

# kayo @ Elpis in ~ [22:49:38] 
$ cat /etc/lsb-release 
───────┬────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: /etc/lsb-release
───────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ DISTRIB_ID=Ubuntu
   2   │ DISTRIB_RELEASE=22.04
   3   │ DISTRIB_CODENAME=jammy
   4   │ DISTRIB_DESCRIPTION="Ubuntu 22.04.1 LTS"
───────┴────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

```

More context here: https://github.com/tauri-apps/tauri/issues/4470#issuecomment-1170342732

**Comment by Sparkenstein** [comment]
Thanks for the report, I looked into this but now I am in a dilemma. Ubuntu 22 will be the future so on one hand I want to support openssl3.0 but on the other hand I am expecting majority of audience will be on prev versions.
at the moment I can provide you a static build that vendors openssl3. 
if you have rust installed then I guess building yourself is pretty straightforward. Otherwise I will look into setting up separate pipeline to build openssl3

**Comment by RyoshiKayo** [comment]
That's fair, I'll build from source for now.

That being said, I think it would probably be a good idea to setup some github action runners to build for the major distros and some sort of release automation. Perhaps we can talk about this on Discord to setup a plan on what to support?

**Comment by Sparkenstein** [comment]
This should be solved by now, I moved GH CI image from 20.04 to 22.04. let me know if issue still persists I don't have a ubuntu machine atm I am on fedora
